### PR TITLE
LRDOCS-5890, now without merge conflicts

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-5890

### CODE REVIEW

**Comment by lipusz** [comment]
**Started reviewing...** :mag:

File: discover/deployment/articles/02-installing-search-engine/01-elasticsearch/00-intro.markdown [hunk 1]
```diff
@@ -1,17 +1,15 @@
 # Configuring Elasticsearch [](id=configuring-elasticsearch-for-liferay-0)
```
Comment (lipusz) [inline discover/deployment/articles/02-installing-search-engine/01-elasticsearch/00-intro.markdown]: Hey Russ

This is now in the "Intro" page. Shouldn't it be called differently? Maybe we can work out this later, but now it talks about configuring and troubleshooting Elasticsearch even before we'd get into the "preparing to install ..." part.

**Comment by lipusz** [comment]
I think, we could refer back to the "discover/deployment/articles/01-deploying-liferay/10-installing-elasticsearch.markdown" page from the "discover/deployment/articles/02-installing-search-engine/01-elasticsearch/01-preparing-to-install.markdown" article (even back and forth) to better guide the audience as these are under two different higher chapters in the guide.

What do you think?

**Comment by rbohl** [comment]
yes, i agree. i can make some progress on that today, and we can improve it more later if needed.

**Comment by lipusz** [comment]
About the "Enterprise Search" articles: I think it'd make sense to rename the pages to "enterprise-search-intro", "enterprise-search-monitoring" and "enterprise-search-security" as ultimately our goal is to promote our products and not the Elastic's features we build on so calling the "x-pack*" maybe not the best choice.

Also, if I consider the new LES offerings some may not really have a direct connection with an Elastic feature, so we'll need to name them differently anyway. So if rename the x-pack ones, we could fit into that direction.

/cc @ccohick fyi

**Comment by lipusz** [comment]
I really like the new structure. Please review my notes. I'll try to respond to you as early as possible. Thanks! :+1:

**Comment by rbohl** [comment]
okay, i'll get on these soon. also, just as an FYI, Jim Hinkey can publish for us if Rich isn't available by the time we get these to him.

**Comment by ccohick** [comment]
> 💡 About the "Enterprise Search" articles: I think it'd make sense to rename the pages to **"enterprise-search-intro", "enterprise-search-monitoring" and "enterprise-search-security"** as ultimately our goal is to promote our products and not the Elastic features we build on so calling them "x-pack*" may not be the best choice.
> 
> Also, if I consider the new LES offerings some may not really have a direct connection with an Elastic feature, so we'll need to name them differently anyway. So if rename the x-pack ones, we could fit into that direction.
> 
> /cc @ccohick fyi

I like the idea for a couple reasons. First, as Tibor said, we may add non-Elastic features to LES. Second, the x-pack naming will go away as we upgrade the supported versions above Elastic 6.3, and the monitoring and security features will be moved within Elasticsearch and Kibana. Our connectors will still be needed, but it saves us keeping up with Elastic's own naming changes.

**Comment by rbohl** [comment]
Suggested edits made in last commit (https://github.com/lipusz/liferay-docs/pull/20/commits/167a67743eb4615f0d11eccf63581a2ffbaee1cf):

**Comment by lipusz** [comment]
Two remarks on the latest changes:
* Please also rename the markdown files for the LES articles. I know it may break existing bookmarks, but otherwise we'll still have pages called "x-pack-monitoring" with title "enterprise search monitoring"
* I don't mind abbreviating Liferay Enterprise Search as LES within the articles, but page titles (main headers) should not contain "LES" as we don't use this form in Help Center article-, offering-, whitepaper titles either so I don't expect so many people to type in and search for "LES Monitoring"

Thank you Russ!

p.s. It's good to know that Jim can also publish. :+1:

**Comment by rbohl** [comment]
Does help center use file names in the UI in some way? LDN does not care about file names. 

Anyway, I updated file names, article titles, and even re-generated headers.

**Comment by lipusz** [comment]
>Does help center use file names in the UI in some way? LDN does not care about file names.

Good point, I'm not sure. But consistency can't hurt. :)

Looks, good thanks everything, Russ! :+1:

**Comment by lipusz** [comment]
https://github.com/sez11a/liferay-docs/pull/3795

**Pull request closed without merging**